client/apps/edit/components/section_text/index.coffee

```diff
@@ -148,7 +148,7 @@ module.exports = React.createClass
           return span { style: {textDecoration: 'line-through'}}
     })(editorState.getCurrentContent())
     # put the line breaks back for correct client rendering
-    html = html.replace('<p></p>', '<p><br></p>').replace('<p> </p>', '<p><br></p>')
+    html = html.replace('<p></p>', '<p><br></p>').replace('<p> </p>', '<p><br></p>').replace('  ', ' &nbsp;')
```

Comment:
would be nice to break this down: http://stackoverflow.com/questions/11195429/coffeescript-and-jquery-chaining